win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1168,22 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    TRACE("Size Image %d, Size Header %d, ResSize %d\n",
+        pbmiCopy->bmiHeader.biSizeImage, pbmiCopy->bmiHeader.biSize, ResSize);
+
+    /* Test if this is a GCC windres.exe compiled 32 bpp bitmap using
+     * BI_BITFIELDS and if so, then a mistake causes it not to include
+     * the bytes for the bitfields. So, we have to substract out the
+     * size of the bitfields previously included from bitmap_info_size.
+     * If this is ever fixed, then this code needs to be removed. */
+    if (compr == BI_BITFIELDS && bpp == 32 &&
+        pbmiCopy->bmiHeader.biSizeImage + pbmiCopy->bmiHeader.biSize == ResSize)
+    {
+        /* GCC pointer to the image data has 12 less bytes than MSVC */
+        pvBits = (char*)pvBits - 12;
+        ERR("GCC Resource Compiled 32-bpp with error\n");
```

Comment:
I am not sure if this warrants an `ERR`, because third-party applications also will have this?

> If this is ever fixed, then this code needs to be removed.

That will never happen, because we need to support third-party applications, and there is no 'automatic' way to detect this fix.

Reply:
I have revised this ERR to make it a WARN now and deleted the comment about "If this is ever fixed,...". Thanks.